subscription-engine/kafka-ws/README.md

```diff
@@ -200,19 +200,20 @@ REDIS_HOST       | Redis host                   | redis
 REDIS_PORT       | Redis port                   | 6379                               | number |
 REDIS_DATABASE   | Redis database               | 1                                  | number |
 
-Note1: Maximum lifetime of a connection will be the highest value between the time calculate from expiration timestamp provided by the jwt token (if KAFKA_WS_JWT_EXP_TIME is true) and KAFKA_WS_MAX_LIFE_TIME. If KAFKA_WS_MAX_LIFE_TIME is -1, it will be the value provided in the time calculate from expiration timestamp provided by the jwt token (if KAFKA_WS_JWT_EXP_TIME is true), otherwise there would be no maximum connection lifetime.
+Note1: Maximum lifetime of a connection will be the highest value between the time calculate from expiration timestamp provided by the JSON Web Token (JWT)  (if KAFKA_WS_JWT_EXP_TIME is true) and KAFKA_WS_MAX_LIFE_TIME. If KAFKA_WS_MAX_LIFE_TIME is -1, it will be the value provided in the time calculate from expiration timestamp provided by the JSON Web Token (JWT)  (if KAFKA_WS_JWT_EXP_TIME is true), otherwise there would be no maximum connection lifetime.
 
-Note2: When KAFKA_WS_JWT_HEADER_AUTH is true, it is checked whether the service (tenant) that is passed in the JWT token can access the kafka topic, generally topics start with `tenant.*`
+Note2: When KAFKA_WS_JWT_HEADER_AUTH is true, it is checked whether the service (tenant) that is passed in the JSON Web Token (JWT) can access the kafka topic, generally topics start with `tenant.*`
 
-### **JWT Token**
+### ** JSON Web Token (JWT) **
 
-When KAFKA_WS_JWT_HEADER_AUTH is true, it is necessary to provide a JWT token in the Header on the Autorization field, as in this example in js below:
+When KAFKA_WS_JWT_HEADER_AUTH is true, it is necessary to provide a JSON Web Token (JWT)  in the Header on the Autorization field, as in this example in js below  (see more https://tools.ietf.org/html/rfc7519#section-4):
```

Comment:
In the context of the dojot, the idea is to use the auth to generate the token, am I right? Otherwise, how do you know it is trustful token?